livebooks/markdown/stylesheets.md

```diff
@@ -350,8 +350,8 @@ We group modifiers together within a class that can be applied to an element. He
 ```elixir
 ~SHEET"""
   "red-title" do
-    foregroundColor(.red);
-    font(.title);
+    foregroundColor(.red)
```

Comment:
this be updated to `foregroundStyle`, right?

Reply:
Yup

Reply:
Done

Reply:
there are two other places, were it remained unchanged.